phylogenetic/rules/write_config.smk

```diff
@@ -0,0 +1,19 @@
+"""
+This part of the workflow writes run time configuration to a YAML file.
+
+OUTPUTS:
+
+    results/run_config.yaml
+"""
+
+rule write_config:
+    output:
+        config = "results/run_config.yaml",
+    log:
+        "logs/write_config.txt",
+    benchmark:
+        "benchmarks/write_config.txt",
+    run:
+        import yaml
+        with open(output.config, 'w') as f:
+            yaml.dump(config, f, sort_keys=False)
```

Comment:
As per https://github.com/nextstrain/WNV/pull/96#discussion_r2244239263, won't `results/run_config.yaml` be overwritten by each snakemake invocation which uses a different configfile? I don't understand why we wouldn't locate this in `results/{build}/config.yaml` (or `run_config.yaml` if preferred)

Reply:
Done (didn't add `build = …` since it's already provided by Snakefile):

https://github.com/nextstrain/WNV/blob/5057a21d3ca6968df7bba0f802d8c2b3f4d78fb2/phylogenetic/rules/write_config.smk#L9-L15

Done for the rest of the workflow: 5057a21d3ca6968df7bba0f802d8c2b3f4d78fb2